tests/test_api/test_types/test_chat.py

```diff
@@ -35,9 +35,14 @@ def test_mention(self, chat_type: Optional[str], title: Optional[str], first_nam
     @pytest.mark.parametrize(
         "chat_type,chat_id,result",
         [
-            ["private", 123124123, 'tg://user?id=123124123']
+            ["private", 123124123, 'tg://user?id=123124123'],
+            ["channel", -10032432324, None]
         ]
     )
     def test_user_url(self, chat_type: Optional[str], chat_id: int, result):
         chat = Chat(type=chat_type, id=chat_id)
-        assert chat.user_url == result
+        if chat_type != 'private':
```

Comment:
Unit tests should not contain any logic. This test should be divided into two different cases, positive and negative, with different sets of input data.